src/main/java/net/atos/zac/app/informatieobjecten/model/validation/ValidRestEnkelvoudigInformatieFileUploadForm.java

```diff
@@ -0,0 +1,26 @@
+package net.atos.zac.app.informatieobjecten.model.validation;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+@Documented
+@Target(TYPE)
+@Retention(RUNTIME)
+@Constraint(validatedBy = {ValidRestEnkelvoudigInformatieFileUploadFormValidator.class})
```

Comment:
Nice! Hope we can get this to work in Kotlin later but let's see.. :-)

Reply:
Yeah, that this might be hard to migrate indeed